quartz_solar_forecast/inverters/enphase.py

```diff
@@ -1,24 +1,43 @@
 import http.client
 import os
+from typing import Optional
+
 import pandas as pd
 import json
 import base64
 from datetime import datetime, timedelta, timezone
 
-from dotenv import load_dotenv
+from urllib.parse import urlencode
 
-load_dotenv()
+from quartz_solar_forecast.inverters.inverter import AbstractInverter
+from pydantic import Field
+from pydantic_settings import BaseSettings
 
-from urllib.parse import urlencode
 
-def get_enphase_auth_url():
+class EnphaseSettings(BaseSettings):
```

Comment:
I think as it's specific to each type of inverter it makes sense to keep these settings models separate. Ideally when someone wants to add a new inverter they shouldn't have to touch too many existing files.

Reply:
> I think as it's specific to each type of inverter it makes sense to keep these settings models separate. Ideally when someone wants to add a new inverter they shouldn't have to touch too many existing files.
> 
> We could pull the inverter models out into separate files, I just think they should be separated by inverter type rather than having everything in `pydantic_models.py`.

I agree with you, the existing files shouldn't be touched when adding a new inverter 
I think having separate directories for each inverter inside the /inverter folder and then further separating the files or folders(up to you) into classes/actual functionalities using those classes